### PR TITLE
improve vertica content

### PR DESCRIPTION
**Pull request by Sachin-Thakur**
## What are you changing in this pull request and why?
 we have made content improvisation on vertica setup and configuration pages 
Describe your changes and why you're making them. If linked to an open
issue or a pull request on dbt Core, then link to them here!

### CODE REVIEW

**Comment by Sachin-Thakur** [comment]
@dataders  could you please review this PR and merge the code base

**Comment by Sachin-Thakur** [comment]
Hi dbt-labs/product-docs  could you please look into the PR

File: website/docs/reference/warehouse-setups/vertica-setup.md [hunk 1]
```diff
@@ -2,22 +2,29 @@
...
   pypi_package: 'dbt-vertica'
-  min_core_version: 'v0.21.0'
+  min_core_version: 'v1.3.0'
   cloud_support: Not Supported
-  min_supported_version: 'Vertica 10.0'
-  slack_channel_name: 'n/a'
-  slack_channel_link: 'https://www.getdbt.com/community/'
+  min_supported_version: 'Vertica v12.0'
+  slack_channel_name: '#adapter-ecosystem'
+  slack_channel_link: 'https://app.slack.com/client/T0VLPD22H/C030A0UF5LM/thread/C030A0UF5LM-1669649490.022339'
```
Comment (dataders) [inline website/docs/reference/warehouse-setups/vertica-setup.md]: this should stay the same. feel free to request a #db-vertica channel in the #channel-requests channel!
```suggestion
  slack_channel_name: 'n/a'
  slack_channel_link: 'https://www.getdbt.com/community/'
```

File: website/docs/reference/warehouse-setups/vertica-setup.md [hunk 2]
```diff
@@ -2,22 +2,29 @@
...
+  authors: Vertica (Former authors:- Matthew Carter, Andy Regan, Andrew Hedengren)
+  github_repo: 'https://github.com/vertica/dbt-vertica/'
   pypi_package: 'dbt-vertica'
-  min_core_version: 'v0.21.0'
+  min_core_version: 'v1.3.0'
   cloud_support: Not Supported
-  min_supported_version: 'Vertica 10.0'
-  slack_channel_name: 'n/a'
-  slack_channel_link: 'https://www.getdbt.com/community/'
+  min_supported_version: 'Vertica v12.0'
```
Comment (dataders) [inline website/docs/reference/warehouse-setups/vertica-setup.md]: the intent of this field is to say:
> to use this adapter, you must have at least this version of Vertica.
so this change tells me that you used to be able to use Vertica 10.0, but cannot anymore. Is that the case?
```suggestion
  min_supported_version: 'Vertica 10.0'
```

File: website/docs/reference/resource-configs/vertica-configs.md [hunk 3]
```diff
@@ -1,28 +1,157 @@
...
 
-It's often desirable to build models as tables in your data warehouse since downstream queries are more performant. While the table materialization also creates your models as tables, it rebuilds the table on each dbt run. These runs can become problematic in that they use a lot of computes when either: 
+#### Configuring the `ignore` (default) parameter
 
-source data tables have millions, or even billions, of rows. 
+<Tabs
+  defaultValue="source"
+  values={[
+    { label: 'Source code', value: 'source', },
+    { label: 'Run code', value: 'run', },
```
Comment (dataders) [inline website/docs/reference/resource-configs/vertica-configs.md]:
```suggestion
    { label: 'Run code', value: 'run', }
```

**Review by dataders — CHANGES_REQUESTED** [review]
some small tweaks! you should be able to look at [the deploy log](https://app.netlify.com/sites/docs-getdbt-com/deploys/63be9226471ffc00082ac641#L94-L103) to find any errors. because there is no preview link, there must be an issue going

File: website/docs/reference/warehouse-setups/vertica-setup.md (hunk 2 above)
Comment (Sachin-Thakur) [reply website/docs/reference/warehouse-setups/vertica-setup.md]: yes @dataders

File: website/docs/reference/warehouse-setups/vertica-setup.md [hunk 4]
```diff
@@ -28,16 +35,27 @@ Some core functionality may be limited. If you're interested in contributing, ch
...
 <h2> Configuring {frontMatter.meta.pypi_package} </h2>
-<p>For dbt-vertica-specifc configuration please refer to <a href={frontMatter.meta.config_page}>{frontMatter.meta.platform_name} Configuration</a> </p>
+
+<p>For {frontMatter.meta.pypi_package} specific configuration please refer to <a href={frontMatter.meta.config_page}>{frontMatter.meta.platform_name} Configuration</a> </p>
+
 <p>For further info, refer to the GitHub repository: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a></p>
-### Connecting to Vertica with **dbt-vertica**
+
+
+### Connecting to Vertica with {frontMatter.meta.pypi_package}
```
Comment (dataders) [inline website/docs/reference/warehouse-setups/vertica-setup.md]: for inane reasons, you can't use markdown headers when injecting front matter info
```suggestion
<h3> Connecting to {frontMatter.meta.platform_name}   with {frontMatter.meta.pypi_package} </h3>
```

**Review by dataders — APPROVED** [review]
thanks for the quick turn around @Sachin-Thakur !

**Comment by Sachin-Thakur** [comment]
> 

Thank You @dataders  for your quick support!

**Pull request merged**